nautobot/dcim/templates/dcim/controller_retrieve.html

```diff
@@ -85,3 +85,8 @@
         </div>
         {% include 'inc/paginator.html' with paginator=devices_table.paginator page=devices_table.page %}
 {% endblock content_full_width_page %}
+
+{% block javascript %}
+{{ block.super }}
+
+{% endblock javascript %}
```

Comment:
Shouldn't be needed, I would think?

Reply:
Let me test, this was copy/paste from `module_retrieve.html`